src/library/crate/cratefeature.cpp

```diff
@@ -277,6 +278,7 @@ bool CrateFeature::activateCrate(CrateId crateId) {
     VERIFY_OR_DEBUG_ASSERT(index.isValid()) {
         return false;
     }
+    emit(unshowTrackModel());
```

Comment:
This reads like an imperative to hide the track model.
But it is actually a kind of hook for a this action. 
"trackModelChanging" or just "saveViewState" or something may work better.

Reply:
done

Reply:
At this point in the call, the sortIndicator may not be selecting to correct column, so it's not correct to use it as the column by which to sort.  Since we just need to trigger a refresh, calling ->select() is sufficient